test/e2e/app-dir/next-after-app-invalid-usage/index.test.ts

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable jest/no-standalone-expect */
+/* eslint-env jest */
+import { nextTestSetup } from 'e2e-utils'
+import * as fs from 'fs'
+import * as path from 'path'
+import * as os from 'os'
+import * as Log from './utils/log'
+import {
+  assertHasRedbox,
+  getRedboxDescription,
+  getRedboxSource,
+} from '../../../lib/next-test-utils'
+
+describe('unstable_after() - invalid usages', () => {
+  const logFileDir = fs.mkdtempSync(path.join(os.tmpdir(), 'logs-'))
+  const logFile = path.join(logFileDir, 'logs.jsonl')
+
+  const { next, isNextDev, skipped } = nextTestSetup({
+    files: __dirname,
+    // `patchFile` and reading runtime logs are not supported in a deployed environment
+    skipDeployment: true,
+    env: {
+      PERSISTENT_LOG_FILE: logFile,
```

Comment:
Is this unused in all three test files? I can't find a reference to this env variable.

Reply:
ah, this is a leftover, will get rid of it